Client/Frontend/Browser/SimpleToast.swift

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+
+struct SimpleToastUX {
+    static let ToastHeight = CGFloat(50)
+    static let ToastAnimationDuration = 0.5
+    static let ToastDefaultColor = UIColor(red: 76.0 / 255, green: 158.0 / 255, blue: 255.0 / 255, alpha: 1)
+    static let ToastFont = UIFont(name: "HelveticaNeue-Thin", size: 14)
+    static let ToastDismissAfter = 0.5
```

Comment:
Probably want to increase this to at least 2 seconds or so -- I find that I barely have time to read it right now.